segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/SegmentStateStore.java

```diff
@@ -31,6 +32,22 @@
 
 /**
  * Stores and Retrieves Segment Attribute values.
+ *
+ * Each state is represented by two files. At least one of these will always be carrying valid data.
+ *
+ * A call to put() will:
+ *  1. If more than one statefiles exist:
+ *      a. Validate that the latest one has correct data.
+ *      b. Delete the older statefile if the latest one has valid data.
+ *  2. Create non-existing statefile write state to it.
+ *  3. Delete the existing statefile.
+ *
+ *  A call to get() will:
+ *  1. Get the latest statefile if more than one statefile exist.
```

Comment:
"if more than one statefile exist." is not a needed clause as it will still get the latest if only one exists.

Reply:
Fixed.